apps/portal/src/content/docs/components/commit-copy-action.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Commit Copy Action
+description: Commit Copy Action component documentation
+---
+
+The Commit Copy Action component provides a UI element for displaying a commit SHA and copying it to the clipboard. It also allows navigation to the commit details page.
+
+import { DocsPage } from "@/components/docs-page";
+
+<DocsPage.ComponentExample
+  client:only
+  code={`
+  <div className="flex">
+    <CommitCopyActions sha="abc1234" toCommitDetails={({ sha }) => "/commit/" + sha} />
+  </div>`}
+/>
+
+## Usage
+
+```typescript jsx
+import { CommitCopyActions } from '@harnessio/ui/components'
+
```

Comment:
Remember the ellipsis to make it obvious that this is not a code-complete example

```suggestion

// ...

```